pkg/httputils/http_utils_test.go

```diff
@@ -2,24 +2,52 @@ package httputils
 
 import (
 	"net/http"
+	"strings"
 	"testing"
 )
 
 func TestBoolValue(t *testing.T) {
-	type args struct {
-		r *http.Request
-		k string
-	}
 	tests := []struct {
-		name string
-		args args
-		want bool
+		name    string
+		testStr string
+		want    bool
 	}{
-		// TODO: Add test cases.
+		{"test normal string", "normal string", true},
+		{"test ''", "", false},
+		{"test 0", "0", false},
+		{"test no", "no", false},
+		{"test No", "No", false},
+		{"test No", "No", false},
+		{"test NO", "NO", false},
+		{"test false", "false", false},
+		{"test False", "False", false},
+		{"test FALSE", "FALSE", false},
+		{"test none", "none", false},
```

Comment:
about the capital test case, I think the NONE and none is enough, no need to add a None or noNe, WDYT?